vim/merlin/autoload/merlin.vim

```diff
@@ -242,7 +242,8 @@ function! merlin#PolaritySearch(debug,query)
   let s:search_result = []
   MerlinPy merlin.vim_polarity_search(vim.eval("a:query"), "s:search_result")
   if a:debug != 1 && s:search_result != []
-    call feedkeys("i=merlin#PolarityComplete()
","n")
+    call feedkeys("i=merlin#PolarityComplete()
+","n")
```

Comment:
I don't think this change should be here.

Reply:
Whoops, fixed.

(to be fair, a stray carriage return in the middle of a line is a bit odd).